.github/workflows/integrational-tests.yml

```
@@ -0,0 +1,35 @@
+name: Integrational tests
+on:
+  push:
+    paths-ignore:
+      - 'doc/**'
+
```

Let's build on pr's also.
```suggestion
  pull_request:
    paths-ignore:
      - 'doc/**'

```